docs/pages/repository.md

```diff
@@ -35,38 +32,76 @@ $repository = $repositoryManager->get(Profile::class);
 
 !!! note
 
-    You can find out more about stores [here](./store.md)
+    The same repository instance is always returned for a specific aggregate.
+
+### Snapshots
 
-### Default Repository Manager
+Loading events for an aggregate is superfast. 
+You can have thousands of events in the database that load in a few milliseconds and build the corresponding aggregate.
 
-// todo
+But at some point you realize that it takes time. To counteract this there is a snapshot store.
```

Comment:
Would add a note about this is behaving like a cache here maybe, or add a warning to make the user aware, even if he is not reading the snapshot doc? Im not sure if needed, just wanted to question it.

Reply:
Just adding the SnapshotStore in DefaultRepositoryManager is not enough to make it cached. You need to read the snapshot store documentation to know how to setup the snapshot store. 

If you have a short sentence without describing much around it, you're welcome. But otherwise i am against much more to describe at this point.